web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -592,105 +504,131 @@ const TeamReviews = ({ onBack, periodId }) => {
     loadReviews();
   }, [loadReviews]);
 
-  const reloadReviews = useCallback(() => {
-    loadedReviews.current = false;
-    loadReviews();
-  }, [loadReviews]);
+  const deleteReviewer = async (member, reviewer) => {
```

Comment:
This is called when the "x" in a reviewer `Chip` is clicked to review the corresponding reviewer from the member described by that row.